sunnyportal/responses.py

```diff
@@ -253,11 +253,13 @@ def parse(self, data):
         self.power_measurements = []
         tag_name = "fiveteen" if self.quarter else "hour"
         for entry in tag.iterfind('./channel/day/%s' % tag_name):
-            mean = self.kw_to_w(entry.get('mean'))
-            if mean is not None:
```

Comment:
Instead of always including all power measurements I added a parameter in 2986ad02 to only include it when requested.